.gitignore

```diff
@@ -10,4 +10,5 @@ node_modules
 reports
 lib
 *.log
+build
```

Comment:
I don't think we ever generate a `build` dir, so there's probably no need to ignore it?

Reply:
removed build